frontend/src/metabase-lib/queries/utils/query-time.js

```diff
@@ -139,46 +143,6 @@ export function generateTimeFilterValuesDescriptions(filter) {
   }
 }
 
-function generateTimeIntervalDescription(n, unit) {
```

Comment:
So apparently this function (almost) always returned an array, unless `!unit && n === 0`, in which case it returns a plain string. This all seemed very wack. It seemed like most the time it got immediately unpacked, or didn't matter.